man/journalctl.xml

```diff
@@ -87,18 +87,15 @@
       causes all matches before and after to be combined in a
       disjunction (i.e. logical OR).</para>
 
-      <para>As shortcuts for a few types of field/value matches, file
-      paths may be specified. If a file path refers to an executable
-      file, this is equivalent to an <literal>_EXE=</literal> match
-      for the canonicalized binary path. Similarly, if a path refers
-      to a device node then match is added for the kernel name of the
-      device (<literal>_KERNEL_DEVICE=</literal>). Also, matches for the
-      kernel names of all the parent devices are added automatically.
```

Comment:
Also:
1. Common English usage supports my choice.
2. The use of the plural "parent devices", is actually misleading, because the user might erroneously infer that a device can have more than one parent and that the parent-child relationship is not part of a tree (as it is), but of a DAG (which it isn't).